### DISCUSSION TITLE
Welcome to Proj-T_C_GEN__Team-77 Discussions!

### DISCUSSION THREAD

**Discussion by Eazyisreal** [zuri-training/Proj-T_C_GEN__Team-77#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amarapeace** [comment]
Hello Eazy, still awake?

**Reply by Eazyisreal** [reply]
I am up. Trying to fix some bugs

**Reply by amarapeace** [reply]
Able group lead, I'm doing my best... You know how it goes, committing and pushing...

**Reply by Eazyisreal** [reply]
It is going well.

**Comment by Eazyisreal** [comment]
Hello amara, how is going with your task?
